From 050f0ebfb65921f5a6f1ae9607c08ef8a25261d5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Karel=20Ko=C4=8D=C3=AD?= <karel.koci@nic.cz>
Date: Wed, 27 Nov 2019 13:53:07 +0100
Subject: [PATCH] Disable flash tab because Turris is autoupdated and this is
 not supported

This only hides it just to make it less available. It still can be
accessed by URL.

---
 .../root/usr/share/luci/menu.d/luci-mod-system.json  | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/modules/luci-mod-system/root/usr/share/luci/menu.d/luci-mod-system.json b/modules/luci-mod-system/root/usr/share/luci/menu.d/luci-mod-system.json
index 90a7001b7..4de4c6517 100644
--- a/modules/luci-mod-system/root/usr/share/luci/menu.d/luci-mod-system.json
+++ b/modules/luci-mod-system/root/usr/share/luci/menu.d/luci-mod-system.json
@@ -110,18 +110,6 @@
 		}
 	},
 
-	"admin/system/flash": {
-		"title": "Backup / Flash Firmware",
-		"order": 70,
-		"action": {
-			"type": "view",
-			"path": "system/flash"
-		},
-		"depends": {
-			"acl": [ "luci-mod-system-flash" ]
-		}
-	},
-
 	"admin/system/reboot": {
 		"title": "Reboot",
 		"order": 90,
-- 
2.26.0

